app/src/main/java/uk/ac/ebi/atlas/experimentpage/json/opentargets/OpenTargetsEvidenceController.java

```diff
@@ -38,22 +38,22 @@
 @Scope("request")
 public class OpenTargetsEvidenceController extends JsonExperimentController {
     private final
-    EvidenceService<DifferentialExpression, DifferentialExperiment, RnaSeqRequestContext, RnaSeqProfile>
+    EvidenceService<DifferentialExpression, DifferentialExperiment, BulkDifferentialRequestContext, BulkDifferentialProfile>
```

Comment:
Rename `diffRnaSeqEvidenceService`  to reflect the change in type. There are also other variables throughout this class with the same issue, please go through it carefully and change all references to `diffRnaSeq` elements.